src/i18n/de/translation.json

```diff
@@ -143,6 +143,11 @@
     "deleteNoteButton": "Kärtchen löschen",
     "deleteStackButton": "Stapel löschen",
     "deleteOptions": "Was möchtest du löschen?",
+    "deleteAccount": "Bist du sicher, dass du dein Konto löschen möchtest?",
+    "deleteAccountText": "Wenn du dein Konto löschst, werden alle damit verbundenen Daten ebenfalls dauerhaft entfernt, einschließlich: \n\n• deiner eigenen erstellten Vorlagen \n• deiner Notizen & Stimmen auf anderen Sitzungen \n\nDiese Aktion kann nicht rückgängig gemacht werden.",
```

Comment:
this works, but isn't the best practice.

maybe consider putting the translations values in an array, then mapping them like this:
```tsx
{t('deleteAccountItems', { returnObjects: true }).map((item) => (
    <li key={item}>{item}</li>
))}
```